pkg/scheduler/objects/application_state.go

```diff
@@ -77,6 +77,7 @@ func (as applicationState) String() string {
 	return [...]string{"New", "Accepted", "Starting", "Running", "Rejected", "Completing", "Completed", "Failing", "Failed", "Expired", "Resuming"}[as]
 }
 
+//nolint:funlen
 func NewAppState() *fsm.FSM {
```

Comment:
could you please do the similar refactor (https://github.com/apache/yunikorn-k8shim/pull/861)?

Reply:
Yes,no problem!